package.json

```diff
@@ -16,8 +16,7 @@
     "version": "yarn version:npm && yarn version:jsr",
     "version:npm": "changeset version",
     "version:jsr": "yarn workspaces foreach --all --no-private --parallel run version:jsr",
-    "version:get": "jq -r .version $INIT_CWD/package.json",
-    "version:set": "jq --arg version $0 '.version = $version' $INIT_CWD/deno.json | sponge $INIT_CWD/deno.json",
+    "version:set": "tmp=$(mktemp) && jq --arg version $0 '.version = $version' $INIT_CWD/deno.json > $tmp && mv $tmp $INIT_CWD/deno.json",
```

Comment:
I don't know why I thought `sponge` was installed in CI.. I must have assumed it was already installed in the ubuntu images